### ISSUE TITLE
Is there are way to generate embeddings for user seq and item seq separately from VQ-rec model 

### ISSUE THREAD

**Issue by shamanez** [RUCAIBox/VQ-Rec#5]
I've seen in the code we mainly use a softmax objective during the fine-tuning.  But I want to use vector similarities (approximate nearest neighbors)  during the inference. 

Have you guys tried the implementation in the above settings?
